docs/source/tutorials/preserve_ram.rst

```diff
@@ -0,0 +1,18 @@
+How to preserve RAM
+===================
+
+The amount of memory consumed can be measured in debug mode with the following lines of code.
```

Comment:
Add environment variable option as well (depends on shell)
Linux `DEPTHAI_LEVEL=debug [myapp]`
Add for bash and powershell

Reply:
Oh ya, I was actually working on that, then forgot about it. Will add this "environmental variables"/"debugging" docs